_changelog/entries/2021/2021-03-15-zoom-v1-fixed-schema-report-meetings-webinars.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Zoom (v1) update: Fixed schema for `report_meetings` and `webinars`
+content-type: "changelog-entry"
+date: 2021-03-15
+entry-type: improvement
```

Comment:
I'd say this should be `updated-feature`. There isn't really an *improvement* here - like nothing about the integration's performance is actually changing. This is just about a changed data type.